packages/govern-console/src/containers/DAO/components/ActionLink/ActionLink.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { useLayout } from '@aragon/ui';
+
+const StyledDiv = styled(NavLink)`
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  height: 48px;
+  padding: 8px 12px;
+  border-radius: 12px;
+
+  font-size: 16px;
+  color: #7483ab;
+  line-height: 20px;
+  font-weight: 600;
+  text-decoration: none;
+
+  /* Active */
+  &.active {
+    color: #00c2ff;
+    background-color: #f0fbff;
+  }
+`;
+
+export type ActionLinkProps = {
+  url: string;
+  label: string;
+  icon: React.ReactElement;
+};
+
+const ActionLink: React.FC<ActionLinkProps> = ({ url, label, icon }) => {
+  const { layoutName } = useLayout();
+
+  return (
+    <StyledDiv
+      to={url}
+      css={layoutName === 'small' ? 'justify-content: center; flex: 1' : undefined}
```

Comment:
Again, here you could extract the `layoutName === 'small'` check into a variable, and then pass it to `StyledDiv` as a prop. Based on that prop you can then move both the `justify-content` and `flex` modification to the styled component itself. 🙂 

(See `.../DaoSideCard.tsx` for more details)